.changeset/dirty-taxis-wonder.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/plugin-techdocs-backend': patch
```

Comment:
Hello @punkle! Shouldn't this be a `minor` version, since the API is changed in a backwards compatible way? 

This is my only comment with regards to techdocs! 👍